service-map/src/main/java/fi/nls/oskari/map/layer/group/link/OskariLayerGroupLinkServiceMybatisImpl.java

```diff
@@ -118,4 +118,11 @@ public boolean hasLinks(int groupId) {
         }
     }
 
+    @Override
+    public List<OskariLayerGroupLink> findByMaplayerId(int layerId){
```

Comment:
This is a duplicate implementation for the  findByLayerId() method in the same file. Remove this one and change callers to use findByLayerId().